scripts/gen_priv_stacks.py

```diff
@@ -101,10 +101,14 @@ def parse_args():
 def main():
     parse_args()
 
+    assert args.kernel, "--kernel ELF required for --gperf-output"
     eh = ElfHelper(args.kernel, args.verbose, kobjects, [])
     syms = eh.get_symbols()
     max_threads = syms["CONFIG_MAX_THREAD_BYTES"] * 8
     objs = eh.find_kobjects(syms)
+    if len(objs) == 0:
```

Comment:
Ditto here.

Reply:
Looks like `objs` isn't used after this point, so could make it `if not eh.find_kobjects(syms):` too. Makes it clear that the result isn't needed beyond that check.

Reply:
Will do.